Sources/CoreAPI/WordPressComOAuthClient.swift

```diff
@@ -187,7 +187,7 @@ public final class WordPressComOAuthClient: NSObject {
             .mapSuccess { response in
                 let responseObject = try JSONSerialization.jsonObject(with: response.body)
 
-                WPKitLogVerbose("Received OAuth2 response: \(self.cleanedUpResponseForLogging(responseObject as AnyObject? ?? "nil" as AnyObject))")
+                // WPKitLogVerbose("Received OAuth2 response: \(self.cleanedUpResponseForLogging(responseObject as AnyObject? ?? "nil" as AnyObject))")
```

Comment:
Might need to introduce a new abstraction layer for logging, too.

Or, is logging at this level useful? Can we simply get rid of it altogether?

Reply:
I opened an issue about it https://github.com/wordpress-mobile/WordPressKit-iOS/issues/787